nexus/lib/qmcpack_converters.py

```diff
@@ -185,6 +185,91 @@ def generate_pw2qmcpack_input(prefix='pwscf',outdir='pwscf_output',write_psir=Fa
 #end def generate_pw2qmcpack_input
 
 
+def import_nexus_module(module_name):
+    import importlib
+    return importlib.import_module(module_name)
+#end def import_nexus_module
+
+
+def read_eshdf_eig_data(filename, Ef_list):
+    import numpy as np
+    from numpy import array,pi
+    from numpy.linalg import inv
+    from unit_converter import convert
+    try:
+        hdfreader = import_nexus_module('hdfreader')
```

Comment:
What case does this style of import help with?

Reply:
idk. I simply copied from `eshdf`. Is it only needed there and not here?

Reply:
In that context is useful to ensure execution of binaries when someone has not taken the step to install Nexus.  I think a normal import is fine here.

Reply:
cool fix in next push